docs/data-sources/modelarts_resource_flavors.md

```diff
@@ -0,0 +1,103 @@
+---
+subcategory: "AI Development Platform (ModelArts)"
+---
+
+# huaweicloud_modelarts_resource_flavors
+
+Use this data source to get resource flavors of ModelArts.
+
+## Example Usage
+
+```hcl
+data "huaweicloud_modelart_resource_flavors" "test" {
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the data source.
+  If omitted, the provider-level region will be used.
+
+* `tag` - (Optional, String) The tag key.
```

Comment:
this is not the tag we commonly use, and this will make users confused
how about replace it with `label0`, and give an example value

Reply:
ok, add parameters to the example